### PR TITLE
Add related links and format title

### PR DESCRIPTION
**Pull request by mccalluc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,10 @@
...
+# Using *Tabula Muris Senis* as reference for a semi-automated scRNA-seq analysis workflow in the cloud
+
+- Tabula Muris
+  - [site](https://tabula-muris.ds.czbiohub.org/)
+  - [github](https://github.com/czbiohub/tabula-muris)
+  - [Single-cell transcriptomics of 20 mouse organs creates a *Tabula Muris* (Nature)](https://www.nature.com/articles/s41586-018-0590-4)
+- Tabula Muris Senis
+  - [site](https://tabula-muris-senis.ds.czbiohub.org/)
+  - [github](https://github.com/czbiohub/tabula-muris-senis)
+  - [A Single Cell Transcriptomic Atlas Characterizes Aging Tissues in the Mouse (BioRxiv)](https://www.biorxiv.org/content/10.1101/661728v2)
```
Comment (aopisco) [inline README.md]: should we also add the link to the s3 bucket with the data?

File: README.md [hunk 2]
```diff
@@ -1 +1,10 @@
-# Using-Tabula-Muris-Senis-as-reference-for-a-semi-automated-sc-RNA-seq-analysis-workflow-in-the-cloud
\ No newline at end of file
+# Using *Tabula Muris Senis* as reference for a semi-automated scRNA-seq analysis workflow in the cloud
```
Comment (aopisco) [inline README.md]: much nicer!

File: README.md (hunk 1 above)
Comment (mccalluc) [reply README.md]: Done! Feel free to merge, if it looks fine.

Comment (aopisco) [reply README.md]: Great!

**Pull request merged**